core-web/libs/dotcms-scss/angular/dotcms-theme/components/form/_treeselect.scss

```diff
@@ -0,0 +1,98 @@
+@use "variables" as *;
+@import "common";
+
+.p-treeselect {
+    @extend #form-field-base;
+    padding: 0;
+    transition: background-color 0.15s, border-color 0.15s, box-shadow 0.15s;
```

Comment:
maybe we can do a constant for this or add it to the mixin if we want this transition on all the form components

Reply:
@zJaaal Not all components share the same attributes to animate therefore in I put the duration like a variable, like this:

`transition: background-color $basic-speed, border-color $basic-speed, box-shadow $basic-speed;`